Packs/FeedMitreAttack/IndicatorFields/incidentfield-mitrecreated.json

```diff
@@ -0,0 +1,30 @@
+{
+ "id": "indicator_mitrecreated",
+ "version": -1,
+ "modified": "2020-09-13T16:40:18.932262+03:00",
+ "name": "Created (technique)",
```

Comment:
why do we have `technique` here? also lets add `MITRE` here so this wont be too generic